reactevalution/src/components/atoms/Icon.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box } from '@mui/material';
+
+interface IconProps {
+  src: string;
+  alt: string;
+}
+
+const Icon: React.FC<IconProps> = ({ src, alt }) => (
+  <Box component="img" src={src} alt={alt} sx={{ width: 24, height: 24 }} />
```

Comment:
don't fix height and width. use styled component or provide className or id to style the component